src/toil_scripts/__init__.py

```diff
@@ -19,3 +21,16 @@ def download_from_s3_url(file_path, url):
         key.get_contents_to_filename(file_path)
     finally:
         s3.close()
+
+
+def generate_file(file_path, generate_func):
```

Comment:
Sorry, missed that earlier. Shall we resolve https://github.com/BD2KGenomics/toil-lib/issues/29 upstream, then propagate that change here?